content_sync/pipelines/concourse.py

```diff
@@ -726,7 +735,8 @@ def upsert_pipeline(self):  # pylint:disable=too-many-locals
             .replace(
                 "((artifacts-bucket))", template_vars["artifacts_bucket_name"] or ""
             )
-            .replace("((ocw-bucket))", template_vars["destination_bucket"] or "")
+            .replace("((web-bucket))", template_vars["web_bucket"] or "")
+            .replace("((offline-bucket))", template_vars["offline_bucket"] or "")
```

Comment:
Why do we default to `""`? Shouldn't we error?

(Even if we should error, I don't think we should address it in this PR, since it seems to be a common pattern in this file.)

Reply:
This is a good point, although the dict of vars is built from env variables in [this function](https://github.com/mitodl/ocw-studio/blob/8895d67e50f9ebb558b97d32239cdfeaccfdda78/content_sync/utils.py#L106). Each pipeline class in `concourse.py` has a `self.MANDATORY_SETTINGS` property [like this](https://github.com/mitodl/ocw-studio/blob/8895d67e50f9ebb558b97d32239cdfeaccfdda78/content_sync/pipelines/concourse.py#L605) that is checked [here](https://github.com/mitodl/ocw-studio/blob/8895d67e50f9ebb558b97d32239cdfeaccfdda78/content_sync/utils.py#L80). So, if any of those settings are not set, an error will be thrown. I added the offline bucket name settings to the `MANDATORY_SETTINGS` list there applicable in this PR.